src/analysis/bounded_variation.lean

```diff
@@ -627,6 +628,44 @@ begin
   exact comp_le_of_monotone_on _ ψ hψ ψts,
 end
 
+namespace set
+
+-- porting note: move to file `data.set.intervals.basic` once the port is over.
+lemma Icc_subsingleton_of_ge {α : Type*} [partial_order α] {a b : α} (h : b ≤ a) :
```

Comment:
```suggestion
lemma subsingleton_Icc_of_ge {α : Type*} [partial_order α] {a b : α} (h : b ≤ a) :
```
so that the lemma name matches the order of words in the statement.